public/app/features/admin/UserOrgs.tsx

```diff
@@ -185,7 +185,7 @@ class UnThemedOrgRow extends PureComponent<OrgRowProps> {
                   orgId={org.orgId}
                   builtInRole={org.role}
                   onBuiltinRoleChange={this.onBuiltinRoleChange}
-                  disabled={rolePickerDisabled}
+                  builtinRolesDisabled={rolePickerDisabled}
```

Comment:
What about the case when !canChangeRole, shouldn't we disable the role picker altogether? not only the builtinRoles?